### PR TITLE
[MLS-1330]: Update AI Training documentation

### PR DESCRIPTION
**Pull request by co42**
Some of the documentation was outdated, some needed more information.

### CODE REVIEW

File: pages/platform/ai-training/14_HOWTO_add_registry/guide.en-gb.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
 ---
-title: Attach a private registry via UI
-slug: attach-private-registry
-excerpt: Learn how to attach your own registry to AI Training via UI
+title: Add a private registry via UI
```
Comment (OVH-Team-Guides) [inline pages/platform/ai-training/14_HOWTO_add_registry/guide.en-gb.md]:
```suggestion
title: Adding a private registry via UI
```

File: pages/platform/ai-training/14_HOWTO_add_registry/guide.en-gb.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
 section: How to (with UI)
 order: 4
 ---
-*Last updated 7th September, 2020.*
+*Last updated 20th April, 2021.*
 
 ## Objective
 
-This guide covers the process of attaching a private registry to the **AI Training** service.
+This guide covers the process of add a private registry to the **AI Training** service.
```
Comment (OVH-Team-Guides) [inline pages/platform/ai-training/14_HOWTO_add_registry/guide.en-gb.md]:
```suggestion
This guide covers the process of adding a private registry to the **AI Training** service.
```

File: pages/platform/ai-training/14_HOWTO_add_registry/guide.en-gb.md [hunk 3]
```diff
@@ -1,20 +1,20 @@
 ---
-title: Attach a private registry via UI
-slug: attach-private-registry
-excerpt: Learn how to attach your own registry to AI Training via UI
+title: Add a private registry via UI
+slug: add-private-registry
```
Comment (OVH-Team-Guides) [inline pages/platform/ai-training/14_HOWTO_add_registry/guide.en-gb.md:3]: As the slug is edited here, should a redirection be added ?

File: pages/platform/ai-training/20_GENERAL_capabilities/guide.en-asia.md [hunk 4]
```diff
@@ -5,22 +5,49 @@ excerpt: Find out about current capabilities and limitations of AI Training
...
-### Jobs resources are limited to:
+### Jobs resources and limitations
 
--   A minimum of 1 GPU and a maximum of 4 GPUs per job
--   No customisable CPU and memory resources
 -   No duration limitation on job execution
 -   No limitation on number of parallel execution
 
+You can either choose the number of GPU or CPU for a job, not both.
+By default, a job use one GPU.
```
Comment (OVH-Team-Guides) [inline pages/platform/ai-training/20_GENERAL_capabilities/guide.en-asia.md]:
```suggestion
By default, a job uses one GPU.
```

File: pages/platform/ai-training/20_GENERAL_capabilities/guide.en-asia.md [hunk 5]
```diff
@@ -5,22 +5,49 @@ excerpt: Find out about current capabilities and limitations of AI Training
...
+-   Memory and local storage resource is not customisable but scaled linearly with each additional CPU.
+
+Information about maximum number of CPU/GPU, memory per CPU/GPU and local storage are available with the `ovhai` CLI.
+
+``` {.console}
+ovhai capability
+```
+
+> [!primary]
+> **Local storage** refer to a drive mounted on the node. It's limited and not the recommended way to handle data, see [OVHcloud documentation on data](../data)
```
Comment (OVH-Team-Guides) [inline pages/platform/ai-training/20_GENERAL_capabilities/guide.en-asia.md]:
```suggestion
> **Local storage** refers to a drive mounted on the node. It's limited and not the recommended way to handle data, see [OVHcloud documentation on data](../data)
```

File: pages/platform/ai-training/20_GENERAL_capabilities/guide.en-asia.md [hunk 6]
```diff
@@ -5,22 +5,49 @@ excerpt: Find out about current capabilities and limitations of AI Training
...
+
 ### Available ports to public network
 
--   HTTP (80, 443, 5000, 8000, 8080, 8443, 9090)
--   Tensorboard (6006)
--   JupyterLab (8888)
+Each job has a public URL, by default this URL access the port 8080 of the job. The default port can be configured when you submit a new job.
```
Comment (OVH-Team-Guides) [inline pages/platform/ai-training/20_GENERAL_capabilities/guide.en-asia.md]:
```suggestion
Each job has a public URL, by default this URL accesses the port 8080 of the job. The default port can be configured when you submit a new job.
```

File: pages/platform/ai-training/21_GENERAL_faq/guide.en-asia.md [hunk 7]
```diff
@@ -34,22 +33,22 @@ Here are the most frequently asked questions about AI Training.
 > > For more information about the failure of a job, start with retrieving the **job** ID with this command: `ovhai job list`
 > >
 > > Once you have your **job** ID, simply retrieve its information with: `ovhai job get <job-id>`
+> > You will see, among others information, the job's state and if the job has run its exit code.
```
Comment (OVH-Team-Guides) [inline pages/platform/ai-training/21_GENERAL_faq/guide.en-asia.md]:
```suggestion
> > You will see, among other information, the job's state and if the job has run its exit code.
```

File: pages/platform/ai-training/21_GENERAL_faq/guide.en-asia.md [hunk 8]
```diff
@@ -34,22 +33,22 @@ Here are the most frequently asked questions about AI Training.
...
+> > There is also the `stateInfo` in which you can evaluate the error message, i.e. whether a command failed or the Docker image was not found.
 > >
 > > For more information you should consult the **job** logs: `ovhai job logs <job-id>`
 > >
-> > The second information you have is the `stateInfo`, in which you can evaluate the error message, i.e. whether a command failed or the Docker image was not found.
-> >
-> My job is in « pending » status, what does it mean ?
-> > You job might be in queue status for 2 main reasons :
+> My job is in « queued » or « pending » state, what does it mean ?
+> > You job might be in this state for 2 main reasons :
```
Comment (OVH-Team-Guides) [inline pages/platform/ai-training/21_GENERAL_faq/guide.en-asia.md]:
```suggestion
> > Your job might be in this state for 2 main reasons :
```

**Review by OVH-Team-Guides — COMMENTED** [review]
A few typos to fix in all versions.
Also we need to know if a redirection needs to be applied.

File: pages/platform/ai-training/14_HOWTO_add_registry/guide.en-gb.md (hunk 3 above)
Comment (co42) [reply pages/platform/ai-training/14_HOWTO_add_registry/guide.en-gb.md:3]: If possible, yes

**Review by OVH-Team-Guides — APPROVED** [review]
Thanks @co42. Redirections will be written tomorrow, along with merge of that PR.

**Comment by OVH-Team-Guides** [comment]
Thanks for your contribution @co42. Merge on Master will happen soon.

**Pull request merged**